verilog/analysis/verilog_linter_test.cc

```diff
@@ -439,6 +439,17 @@ TEST(VerilogLinterDocumentationTest, AllRulesMarkdown) {
   EXPECT_TRUE(absl::StrContains(stream.str(), "Enabled by default:"));
 }
 
+TEST(VerilogLinterDocumentationTest, PrintLintRuleFile) {
+  std::ostringstream stream;
+  verilog::GetLintRuleFile(&stream);
+  // Spot-check a few patterns, must mostly make sure generation
+  // works without any fatal errors.
+  EXPECT_TRUE(absl::StrContains(stream.str(), "+always-comb"));
+  EXPECT_TRUE(absl::StrContains(
+      stream.str(), "+module-filename=allow-dash-for-underscore:false"));
+  EXPECT_TRUE(absl::StrContains(stream.str(), "-forbid-negative-array-dim"));
```

Comment:
A good test would be to do a roundtrip
  * have a set of rules that we print out
  * use the print-out and parse that into a new set of rules. This will make sure it is parseable.
  * Test that the newly parsed set of rules matches the original set of rules, i.e. that the parsing was successful in extracting all the data.

Reply:
Yep, I'll put something together.

Reply:
I've implemented this change. Let me know what you think.